docs/Release-Notes-2.16.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: default
+title: Release Notes 2.16
+---
+
+Released on XXX.
+
+## Mock-core-configs 36.4
+
+The biggest change is the removal of `epel-8-*` configs. It has been replaced by several configs: `alma+epel-8-*`, `centos+epel-8-*`, `oraclelinux+epel-8-*`, `rhel+epel-8-*`, `rocky+epel-8-*`. Every config has its pros and cons:
+
+* `alma+epel-8-*` - This uses Alma Linux 8 + Fedora EPEL. It works right off the bat and it is recommended replacement. The only disadvantage is that Koji actually uses RHEL + EPEL for EPEL builds.
+* `centos+epel-8-*` - This uses CentOS 8 + Fedora EPEL. We do **not** recommend to use this config, because [CentOS 8 will reach EOL on December 31, 2021(https://www.centos.org/centos-linux-eol/).
+* `oraclelinux+epel-8-*` - This uses Oracle Linux 8 + Fedora EPEL.
+* ``rhel+epel-8-*` - This uses RHEL 8 + Fedora EPEL. This is the configuration that Koji uses. But it [requires some settings](Feature-rhelchroots) and RHEL subscriptions. As developer, you can have (16 subscriptions for free)[https://developers.redhat.com/blog/2021/02/10/how-to-activate-your-no-cost-red-hat-enterprise-linux-subscription]
```

Comment:
2 markdown typos:

```suggestion
* `rhel+epel-8-*` - This uses RHEL 8 + Fedora EPEL. This is the configuration that Koji uses. But it [requires some settings](Feature-rhelchroots) and RHEL subscriptions. As developer, you can have [16 subscriptions for free](https://developers.redhat.com/blog/2021/02/10/how-to-activate-your-no-cost-red-hat-enterprise-linux-subscription)
```

Reply:
Also not sure if the `Feature-rhelchroots` link works, it certainly does not when viewing the rendered document on GitHub.